mod/wikirate/set/type/wikirate_company.rb

```diff
@@ -7,10 +7,10 @@
 
 def indirect_contributor_search_args
   [
-    {:type_id=>ClaimID, :plus=>['company',:link_to=>self.name]},
-    {:type=>'source', :plus=>['company',:link_to=>self.name]},
-    {:type=>'analysis', :left=>self.name },
-    {:type=>'metric value', :left=>{:right=>self.name}}
+    {:type_id=>Card::ClaimID, :plus=>['company',:link_to=>self.name]},
+    {:type_id=>Card::SourceID, :plus=>['company',:link_to=>self.name]},
+    {:type_id=>Card::WikirateAnalysisID, :left=>self.name },
+    {:type_id=>Card::MetricValueID, :left=>{:right=>self.name}}
   ]
```

Comment:
You don't need it in set modules, because when the module is created, the code is in:

````
class Card
  module Set
    module Type
    ....
    end
  end
end

In other words, you're already in the context of `Card`.  If you're outside of that context (eg in a rake task), you need the full name.